packages/datagrid/src/components/DefaultHeaderRenderer/MenuDropDownQualityBar.component.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Overlay, Popover, Button } from 'react-bootstrap';
+import { Action } from '@talend/react-components';
+import './MenuDropDownQualityBar.scss';
+
+
+export default function MenuDropDownQualityBar(props) {
+	const [isOpen, setIsOpen] = React.useState(false);
+	const buttonRef = React.useRef();
+	return (<>
+		<Action
+			buttonRef={buttonRef}
+			className={'button-menu-dropdown'}
+			data-feature={props.dataFeature}
+			label={props.label}
+			link
+			hideLabel
+			onClick={() => {
```

Comment:
Don't we have overlay already handle by the Action component instead of doing it manually ?

Reply:
yes, it's right but we have lot of css in this component, and i need to modify lot of this css if i take this component (i already try on TDP on other tickets)